console/console-init/ui/mock-console-server/mock-console-server.js

```diff
@@ -527,28 +595,35 @@ function createAddressSpace(as) {
   var phase = "Active";
   var messages = [];
   if (as.status && as.status.phase) {
-    phase = as.status.phase
+    phase = as.status.phase;
   }
   if (phase !== "Active") {
-    messages.push("The following deployments are not ready: [admin.daf7b31]")
+    messages.push("The following deployments are not ready: [admin.daf7b31]");
   }
 
   var addressSpace = {
     metadata: {
       name: as.metadata.name,
       namespace: namespace.metadata.name,
       uid: uuidv1(),
-      creationTimestamp: as.metadata.creationTimestamp ? as.metadata.creationTimestamp : getRandomCreationDate()
+      creationTimestamp: as.metadata.creationTimestamp
+        ? as.metadata.creationTimestamp
+        : getRandomCreationDate()
     },
     spec: {
       plan: spacePlan,
       type: as.spec.type,
       endpoints: as.spec.endpoints ? as.spec.endpoints : createDefaultEndpoints(),
       authenticationService: {
-        name: as.spec.authenticationService ? as.spec.authenticationService.name : null
+        name: as.spec.authenticationService
+          ? as.spec.authenticationService.name
+          : null
       }
     },
-    status: null
```

Comment:
@k-wall I made a small change in the mock server as it is not matching with exact behaviour of real backend.